discord_bot/discord_bot.py

```diff
@@ -4,41 +4,10 @@
 import websockets
 
 from .opcodes import Opcodes
+from .events import Events
+from ._version import __version__
```

Comment:
We should probably move the version file to the top level of the repository, since it's the version of the codebase, not the `discord_bot` module. The import statement will need to change though.